extensions-contrib/opentelemetry-emitter/pom.xml

```diff
@@ -169,7 +169,6 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
-        <version>3.2.4</version>
```

Comment:
is this now part of the main pom?

Reply:
Yeah! looks like Ivan made that change already in the main pom. [Reference](https://github.com/confluentinc/druid/blob/0.22.1-confluent/pom.xml#L1591)
The issue I hit seemed to be an intermittent one. So, this line didn't really do anything :)